model/build.gradle

```diff
@@ -23,7 +23,9 @@ protobuf {
 }
 
 dependencies {
-  implementation 'com.google.protobuf:protobuf-lite:3.0.0'
+  // This must use compile in order for lite protobufs to generate correctly.
+  // TODO(#59): Move to Bazel to avoid relying on a deprecated Gradle directive.
+  compile 'com.google.protobuf:protobuf-lite:3.0.0'
```

Comment:
Is this change still needed after you merge in the latest develop branch? I'm surprised that this would be needed since the splash page doesn't use any protos.